test/ScalarImageToRunLengthFeaturesImageFilterInstantiationTest.cxx

```diff
@@ -22,59 +22,94 @@
+    return EXIT_FAILURE;
+    }
 
+  const unsigned int ImageDimension = 3;
+  const unsigned int VectorComponentDimension = 10;
 
+  // Declare types
+  typedef int                                         InputPixelType;
+  typedef float                                       OutputPixelComponentType;
+  typedef itk::Vector< OutputPixelComponentType, VectorComponentDimension >
+                                                      OutputPixelType;
+
+  typedef itk::Image< InputPixelType, ImageDimension >  InputImageType;
+  typedef itk::Image< OutputPixelType, ImageDimension > OutputImageType;
+  typedef itk::ImageFileReader< InputImageType >        ReaderType;
+  typedef itk::Neighborhood< typename InputImageType::PixelType,
+    InputImageType::ImageDimension >                    NeighborhoodType;
+
+
+  // Create and set up a reader
+  ReaderType::Pointer reader = ReaderType::New();
+  reader->SetFileName( argv[1] );
+
+  // Create and set up a maskReader
+  ReaderType::Pointer maskReader = ReaderType::New();
+  maskReader->SetFileName( argv[2] );
+
+  // Create the filter
+  typedef itk::Statistics::ScalarImageToRunLengthFeaturesImageFilter<
+    InputImageType, OutputImageType > FilterType;
+
+  FilterType::Pointer filter = FilterType::New();
+
+  EXERCISE_BASIC_OBJECT_METHODS( filter,
+    ScalarImageToRunLengthFeaturesImageFilter, ImageToImageFilter );
+
+
+  filter->SetInput( reader->GetOutput() );
+
+  filter->SetMaskImage( maskReader->GetOutput() );
+  TEST_SET_GET_VALUE( maskReader->GetOutput(), filter->GetMaskImage() );
+
+  unsigned int numberOfBinsPerAxis = 15;
+  filter->SetNumberOfBinsPerAxis( numberOfBinsPerAxis );
+  TEST_SET_GET_VALUE( numberOfBinsPerAxis, filter->GetNumberOfBinsPerAxis() );
+
+  FilterType::PixelType pixelValueMin = -62;
+  FilterType::PixelType pixelValueMax = 2456;
+  filter->SetPixelValueMinMax( pixelValueMin, pixelValueMax );
+  TEST_SET_GET_VALUE( pixelValueMin, filter->GetMin() );
+  TEST_SET_GET_VALUE( pixelValueMax, filter->GetMax() );
+
+  FilterType::RealType minDistance = 0.15;
+  FilterType::RealType maxDistance = 1.5;
+  filter->SetDistanceValueMinMax( minDistance, maxDistance );
+  TEST_SET_GET_VALUE( minDistance, filter->GetMinDistance() );
+  TEST_SET_GET_VALUE( maxDistance, filter->GetMaxDistance() );
+
+  NeighborhoodType::SizeValueType neighborhoodRadius = 3;
+  NeighborhoodType hood;
+  hood.SetRadius( neighborhoodRadius );
+  filter->SetNeighborhoodRadius( hood.GetRadius() );
+  TEST_SET_GET_VALUE( hood.GetRadius(), filter->GetNeighborhoodRadius() );
+
+  FilterType::PixelType insidePixelValue = 0;
+  filter->SetInsidePixelValue( insidePixelValue );
+  TEST_SET_GET_VALUE( insidePixelValue, filter->GetInsidePixelValue() );
+
+  FilterType::OffsetType offset = {{-1, 0, 1}};
+  FilterType::OffsetVector::Pointer offsetVector = FilterType::OffsetVector::New();
+  offsetVector->push_back( offset );
+  TEST_SET_GET_VALUE( offsetVector, filter->GetOffsets() );
```

Comment:
This line is making the test fail: indeed there is no GetOffsets() function implemented in ScalarImageToRunLengthFeaturesImageFilter for now.

I will work on repairing this mistake by adding this missing function before to merge this patch.